### PR TITLE
Enhance trying out the language page

### PR DESCRIPTION
**Pull request by jbampton**
Run pre-commit

### CODE REVIEW

**Review by johnbampton — DISMISSED** [review]

**Review by amir-shiati — APPROVED** [review]

File: en/trying-out-the-language.md [hunk 1]
```diff
@@ -16,16 +16,16 @@ It's here:
 
 ## Trying on your system
 
-For this case, you first need to install One compiler, currently, You are able to compile the One compiler in Linux (both x86_64 and i386 architecture) and macOS, and BSD family, and finally Windows OS.
```
Comment (pedramcvx1) [inline en/trying-out-the-language.md:19]: For this case, you first need to install One compiler. **Currently you** are able to compile the One compiler **on** Linux (both x86_64 and i386 architecture), **macOS, the BSD family, and finally Windows OS.**

**Review by pedramcvx1 — CHANGES_REQUESTED** [review]
other than that, LGTM.

File: en/trying-out-the-language.md [hunk 2]
```diff
@@ -16,16 +16,16 @@ It's here:
...
+For this case, you first need to install the One compiler. You are able to compile the One compiler in Linux (both x86_64 and i386 architecture) and macOS, the BSD family, and finally Windows OS.
```
Comment (jbampton) [inline en/trying-out-the-language.md:19]:
```suggestion
For this case, you first need to install the One compiler. You are able to compile the One compiler in Linux (both x86_64 and i386 architecture), macOS, the BSD family, and finally Windows OS.
```

**Review by johnbampton — APPROVED** [review]

**Review by pedramcvx1 — APPROVED** [review]
LGTM

**Comment by jbampton** [comment]
Changed just now to one approving green review to merge.

**Pull request merged**